about.html

```diff
@@ -10,55 +10,67 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
-<body>
+<div class="top-container">
+  <header>
+      <nav>
```

Comment:
You have your alt="" here but you should also fill it out so your page is more accessible.